angularSTEP/src/app/app.module.ts

```diff
@@ -69,9 +70,13 @@ import {ViewProfilesComponent} from './view-profiles/view-profiles.component';
     RecipesComponent,
     RecipePageComponent,
     CurrentProfilePageComponent,
-    MyProfileTabComponent
+    MyProfileTabComponent,
```

Comment:
Not part of your changes but for completeness: Names like MyFoo should usually not go into the final non-test code.

Reply:
Oops, yes that was me actually. I will change the name! It made sense to me at the time but you are right.